testplan/runnable/interactive/base.py

```diff
@@ -726,71 +726,6 @@ def all_tests_operation(self, operation, await_results=True):
             else:
                 raise ValueError("Unknown operation: {}".format(operation))
 
-    def create_new_environment(self, env_uid, env_type="local_environment"):
-        """Dynamically create an environment maker object."""
-        if env_uid in self._created_environments:
-            raise RuntimeError(
-                "Environment {} already exists.".format(env_uid)
-            )
-
-        if env_type == "local_environment":
-            from testplan.environment import LocalEnvironment
-
-            env_class = LocalEnvironment
-        else:
-            raise ValueError("Unknown environment type: {}".format(env_type))
-
-        self._created_environments[env_uid] = env_class(env_uid)
-
-    def add_environment_resource(
-        self, env_uid, target_class_name, source_file=None, **kwargs
-    ):
-        """
-        Add a resource to existing environment or to environment maker object.
-        """
-        final_kwargs = {}
-        compiled = re.compile(r"_ctx_(.+)_ctx_(.+)")
-        context_params = {}
-        for key, value in kwargs.items():
-            if key.startswith("_ctx_"):
-                matched = compiled.match(key)
-                if not matched or key.count("_ctx_") != 2:
-                    raise ValueError("Invalid key: {}".format(key))
-                target_key, ctx_key = matched.groups()
-                if target_key not in context_params:
-                    context_params[target_key] = {}
-                context_params[target_key][ctx_key] = value
-            else:
-                final_kwargs[key] = value
-        if context_params:
-            from testplan.common.utils.context import context
-
-            for key in context_params:
-                final_kwargs[key] = context(**context_params[key])
-
-        if source_file is None:  # Invoke class loader
-            resource = self._resource_loader.load(
-                target_class_name, final_kwargs
-            )
-            try:
-                self.get_environment(env_uid).add(resource)
-            except:
-                self._created_environments[env_uid].add_resource(resource)
-        else:
-            raise Exception("Add from source file is not yet supported.")
-
-    def reload_environment_resource(
-        self, env_uid, target_class_name, source_file=None, **kwargs
-    ):
-        # Placeholder for function to delele an existing and registering a new
-        # environment resource with probably altered source code.
-        # This should access the already added Environment to plan.
-        pass
-
-    def add_created_environment(self, env_uid):
-        """Add an environment from the created environment maker instance."""
-        self.target.add_environment(self._created_environments[env_uid])
-
```

Comment:
All kinds of unused functions, we can restore from history if needed at some point.